js/scale.js

```diff
@@ -0,0 +1,52 @@
+const uploadForm = document.querySelector('.img-upload__form');
+const reduceButton = uploadForm.querySelector('.scale__control--smaller');
+const increaseButton = uploadForm.querySelector('.scale__control--bigger');
+const scaleValue = uploadForm.querySelector('.scale__control--value');
+const picturePreview = uploadForm.querySelector('.img-upload__preview');
+
+const pictureSize = {
```

Comment:
```suggestion
const PictureSize = {
```
енамы пишутся с большой буквы